bin/handlerFunctions.js

```diff
@@ -1314,7 +1286,10 @@ async function getFiles(path) {
 // check if config exists function
 function checkConfigExists() {
     let exists;
-    if (!fs.existsSync(CONFIG_FILE_PATH)) {
+    let configFilePath = getConfigPath();
+
+    console.log(`checkConfigExists() configFilePath: ${configFilePath}`);
```

Comment:
Doesn't look like this is needed, since this is already logged in the getConfigPath method.